src/main/java/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't load JDBC driver for mySQL", e);
+        }
+    }
+
+    public static Connection getConnection() {
+        try {
+            Properties dbProperties = new Properties();
+            dbProperties.put("user", "root");
+            dbProperties.put("password", "1234567");
```

Comment:
```suggestion
            dbProperties.put("user", USERNAME);
            dbProperties.put("password", PASSWORD);
```
use constants so you can quickly change value in just one place (it will be handy later when you'll have bigger projects)
also jdbc driver name can also be constant